### PR TITLE
Hotfix: 4.24.2

### PR DESCRIPTION
**Pull request by vault087**
- User-Agent's SDK Version Number is fixed
- Billing Address's country selector presents modally if no navigation controller found

### CODE REVIEW

File: OmiseSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1558,7 +1558,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 4.24.0;
```
Comment (aashishgurung) [inline OmiseSDK.xcodeproj/project.pbxproj:1561]: We don't need to include `MARKETING_VERSION` anymore?

**Review by ajzkk — APPROVED** [review]

File: OmiseSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (danfowler) [reply OmiseSDK.xcodeproj/project.pbxproj:1561]: @aashishgurung I think the idea is we are replacing this version string with the version string set in `OmiseSDK/Client.swift`

File: OmiseSDK/CreditCardFormViewController.swift [hunk 2]
```diff
@@ -203,7 +203,7 @@ public class CreditCardFormViewController: UIViewController, PaymentChooserUI, P
     }
 
     @available(*, deprecated,
-                message: "Please use the new method that confrom to Objective-C convention +[OMSCreditCardFormViewController creditCardFormViewControllerWithPublicKey:] as of this method will be removed in the future release.", // swiftlint:disable:this line_length
+                message: "Please use the new method that confrom to Objective-C convention +[OMSCreditCardFormViewController creditCardFormViewControllerWithPublicKey:] as of this method will be removed in the future release.",
```
Comment (danfowler) [inline OmiseSDK/CreditCardFormViewController.swift:206]:
```suggestion
                message: "Please use the new method that conforms to Objective-C convention +[OMSCreditCardFormViewController creditCardFormViewControllerWithPublicKey:] as this method will be removed in a future release.",
```

File: OmiseSDK/CreditCardFormViewController.swift [hunk 3]
```diff
@@ -186,7 +186,7 @@ public class CreditCardFormViewController: UIViewController, PaymentChooserUI, P
     @objc public static let defaultErrorMessageTextColor = UIColor.error
 
     /// A boolean flag that enables/disables Card.IO integration.
-    @available(*, unavailable, message: "Built in support for Card.ios was removed. You can implement it in your app and call the setCreditCardInformation(number:name:expiration:) method") // swiftlint:disable:this line_length
+    @available(*, unavailable, message: "Built in support for Card.ios was removed. You can implement it in your app and call the setCreditCardInformation(number:name:expiration:) method")
```
Comment (danfowler) [inline OmiseSDK/CreditCardFormViewController.swift:189]:
```suggestion
    @available(*, unavailable, message: "Built-in support for Card.ios was removed. You can implement it in your app and call the setCreditCardInformation(number:name:expiration:) method")
```

**Review by danfowler — APPROVED** [review]

File: OmiseSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (vault087) [reply OmiseSDK.xcodeproj/project.pbxproj:1561]: Somehow Swift Package Manager updates Info.plist file and all information about versions gone making this MARKETING_VERSION useless for the Framework.

Comment (vault087) [reply OmiseSDK.xcodeproj/project.pbxproj:1561]: In the future I'll check how to arrange this in a better way (if possible) but at the moment don't want to waste the time doing it.

**Pull request merged**